inclusion_connect/accounts/views.py

```diff
@@ -284,3 +294,10 @@ def form_valid(self, form):
         login(self.request, self.get_object())
         messages.success(self.request, "Votre mot de passe a été mis à jour.")
         return super().form_valid(form)
+
+
+def return_to_referrer(request):
+    referrer_uri = request.session.pop("referrer_uri", None)
+    if not referrer_uri:
+        return Http404
+    return HttpResponseRedirect(referrer_uri)
```

Comment:
Ça revient au même, mais évite la négation.
```suggestion
    if referrer_uri:
        return HttpResponseRedirect(referrer_uri)
    return Http404
```